flutter_local_notifications/example/lib/main.dart

```diff
@@ -316,6 +316,14 @@ class _HomePageState extends State<HomePage> {
                         await _scheduleDailyTenAMNotification();
                       },
                     ),
+                    PaddedRaisedButton(
+                      buttonText:
+                          'Schedule daily in 5 seconds last year notification '
```

Comment:
Can you check the text and method name to say it'll schedule a daily 10:00 am notification similar to the other examples? The reason for this is that it's more difficult to figure out the time with the example that is there now. Part of this is due how it references 5 seconds in the future but uses a date last year. Expressing this as an actual time makes it more clearer that the time is constant as that is people usually try to schedule something that is recurring. As an example, in normal conversation one is more likely to say they go to the gym everyday at 7:00 AM instead as opposed to saying they go to the gym everyday in 5 seconds

Reply:
I understand, but the reason i choose "in 5 seconds", is so that we are actually able to test it, and don't have to wait for the next 10:00 am to actually see the notification. Should not people actually be able to test / see the notification in the examples?

Reply:
They can but it wasn't meant to be immediate for the same reasons I explained earlier regarding "communication" and how recurring notifications/events are usually scheduled. Given, it's recurring, users still actually need to wait to validate that it repeats. In <x amount of time> requires a bit of calculation. Developers can still change the code to suit the time if needed

Reply:
Ok, but i should still note that it's last year right?

Reply:
Yep so the text and method name would refer to something along the lines of "Schedule daily 10:00:00AM notification using last year's date"